merino/curated_recommendations/localization.py

```diff
@@ -0,0 +1,72 @@
+"""Hardcoded localized strings."""
+
+from merino.curated_recommendations.corpus_backends.protocol import ScheduledSurfaceId
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+LocalizedTopicSectionTitles = dict[ScheduledSurfaceId, dict[str, str]]
+
+# Hardcoded localized topic section titles (for now) for en-US & de-DE
+LOCALIZED_TOPIC_SECTION_TITLES: LocalizedTopicSectionTitles = {
```

Comment:
top-stories is not a topic section, so a better name might be:
```suggestion
LOCALIZED_SECTION_TITLES: LocalizedSectionTitles = {
```
I can also do this when adding the News Section later this week.

Reply:
updated this in my PR along with the German string for top-stories